### PR TITLE
ADD: Colaborator on migration to V12

### PR DESCRIPTION
**Pull request by flectra-chile**
Please take into account our contributions at repository base 
https://gitlab.com/dansanti/l10n_cl_fe/commit/c24974026aa23331757641df720c75e441ce3f1f

CC @nelsonramirezs

### CODE REVIEW

**Comment by max3903** [comment]
Ping @nelsonramirezs

File: l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -2,3 +2,4 @@
 * Nelson Ramirez
 * Carlos Toledo
 * Carlos Lopez
+* Flectra Chile
```
Comment (max3903) [inline l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst]: @flectra-chile We will need firstname and lastname of individuals here with email address if possible. Thank you.

**Review by max3903 — CHANGES_REQUESTED** [review]

**Comment by nelsonramirezs** [comment]
> Ping @nelsonramirezs
@max3903  That commit is not part of this module.
https://gitlab.com/dansanti/l10n_cl_fe/commit/c24974026aa23331757641df720c75e441ce3f1f
@flectra-chile 
But help is always needed.

File: l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (Danisan) [reply l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst]: @flectra-chile the name itself is confrontative and conflictive in my opinion

Comment (flectra-chile) [reply l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst]: > @flectra-chile We will need firstname and lastname of individuals here with email address if possible. Thank you.

@max3903  Flectra Chile is SpA, individual contributor is Francisco Lorca <florca1979@gmail.com>

I Update PR for avoid confusion

File: l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -2,3 +2,4 @@
 * Nelson Ramirez
 * Carlos Toledo
 * Carlos Lopez
+* Francisco Lorca
```
Comment (max3903) [inline l10n_cl_electronic_invoicing/readme/CONTRIBUTORS.rst]:
```suggestion
* Flectra Chile <https://flectrachile.cl/>

  * Francisco Lorca <florca1979@gmail.com>
```

**Comment by max3903** [comment]
@flectra-chile Can you take care of signing and sending the OCA CLA please?

**Comment by flectra-chile** [comment]
> @flectra-chile Can you take care of signing and sending the OCA CLA please?

Entity CLA?

**Comment by max3903** [comment]
> > @flectra-chile Can you take care of signing and sending the OCA CLA please?
> 
> Entity CLA?

Yes, if you want to cover your colleagues.

**Pull request closed without merging**